.github/workflows/stream_core_flutter_workflow.yml

```diff
@@ -0,0 +1,74 @@
+name: stream_core_flutter_workflow
+
+env:
+  FLUTTER_CHANNEL: stable
+  FLUTTER_VERSION: 3.29.0
+  ENV_PROPERTIES: ${{ secrets.ENV_PROPERTIES }}
+
+on:
+  pull_request:
+    types:
+      - opened
+      - reopened
+      - synchronize
+      - ready_for_review
+  push:
+    branches:
+      - main
+
+concurrency:
+  group: ${{ github.workflow }}-${{ github.ref }}
+  cancel-in-progress: true
+
+jobs:
+  analyze:
+    timeout-minutes: 15
+    runs-on: ubuntu-latest
+    steps:
+      - name: Git Checkout
+        uses: actions/checkout@v3
+        with:
+          fetch-depth: 0
+
+      - name: Install Flutter
+        uses: subosito/flutter-action@v2
+        with:
+          cache: true
+          channel: ${{ env.FLUTTER_CHANNEL }}
+          flutter-version: ${{ env.FLUTTER_VERSION }}
+
+      - name: Install Tools
+        run: |
+          flutter pub global activate melos
+          flutter pub global activate remove_from_coverage
+
+      - name: Bootstrap Workspace
+        run: melos postclean
```

Comment:
Is this step required?

Reply:
Wouldn't think so, I got it from the video repo, but will remove it.